lib/internal/v8_prof_processor.js

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable strict */
 const scriptFiles = [
   'internal/v8_prof_polyfill',
-  'v8/tools/splaytree',
-  'v8/tools/codemap',
-  'v8/tools/csvparser',
-  'v8/tools/consarray',
-  'v8/tools/profile',
-  'v8/tools/profile_view',
-  'v8/tools/logreader',
-  'v8/tools/tickprocessor',
-  'v8/tools/SourceMap',
-  'v8/tools/tickprocessor-driver'
+  v8_parent_path + '/tools/splaytree',
+  v8_parent_path + '/tools/codemap',
+  v8_parent_path + '/tools/csvparser',
+  v8_parent_path + '/tools/consarray',
+  v8_parent_path + '/tools/profile',
+  v8_parent_path + '/tools/profile_view',
+  v8_parent_path + '/tools/logreader',
+  v8_parent_path + '/tools/tickprocessor',
+  v8_parent_path + '/tools/SourceMap',
+  v8_parent_path + '/tools/tickprocessor-driver'
```

Comment:
Need to declare and assign `v8_parent_path` -- see https://github.com/ibmruntimes/node/blob/v4.x-port/lib/internal/v8_prof_processor.js#L2